### ISSUE TITLE
Arch; lsof dependency

### ISSUE THREAD

**Issue by imunolion** [keybase/keybase-issues#2427]
I'm using Arch Linux and installed app from yaourt.
I found some commands require me `lsof` like `keybase update check-in-use`. However, my yaourt didn't manage to install it. Maybe because of missing dependency there??

**Comment by imunolion** [comment]
Yes, according to AUR, keybase-bin depends on only fuse and gconf.

**Comment by strib** [comment]
@oconnor663: yeah we should probably add a dependency for `lsof`.

**Comment by oconnor663** [comment]
Thanks for the report. I've updated `keybase-git` and `keybase-bin`. I'll also want to add this dependency to our deb/rpm packages, even though Ubuntu at least seems to have lsof installed by default. (Which is probably why we haven't heard about this before now.)

@strib, can you think of any other external dependencies that KBFS would have?

**Comment by oconnor663** [comment]
[internal task](https://keybase.atlassian.net/browse/CORE-3484)

@strib, also, what's the dependency situation on OSX and Windows?

**Comment by strib** [comment]
Thanks @oconnor663!

> @strib, can you think of any other external dependencies that KBFS would have?

Technically this isn't a KBFS dependency, it's an updater dependency.  KBFS itself shouldn't have any besides fuse as far as I know, but I can't speak for the updater.

> @strib, also, what's the dependency situation on OSX and Windows?

I'm the wrong person to ask.  @zanderz, @gabriel?

**Comment by oconnor663** [comment]
@gabriel looks like `lsof` is a client dependency, but only for commands relevant to the updater (which we don't ship on Linux). Would it make more sense to keep _not_ depending on lsof, and maybe even disable these commands on Linux?

**Comment by gabriel** [comment]
The lsof package doesn't have any build platform directives... It will throw an error if `/usr/sbin/lsof` doesn't exist on exec.Command.

We only use `keybase update check-in-use` for macOS at the moment so you could ignore that this errors on other platforms, or we could change the error message to be "Unsupported for your platform" after checking runtime.GOOS != "darwin".

**Comment by oconnor663** [comment]
Well the good news is I foolishly edited some generated files to add the `lsof` dependency two days ago, so the changes automatically got wiped out with the next daily build :|
